docker/Dockerfile

```diff
@@ -24,13 +24,15 @@ RUN useradd -m -s /bin/bash -N -u $NB_UID $NB_USER && \
 USER keras
 
 # Python
-ARG python_version=3.5.2
-ARG tensorflow_version=0.12.0rc0-cp35-cp35m
+ARG python_version=3.5
+ARG tensorflow_version=1.0.1-cp35-cp35m
+ARG theano_version=0.9
 RUN conda install -y python=${python_version} && \
+    pip install --upgrade pip && \
     pip install https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow_gpu-${tensorflow_version}-linux_x86_64.whl && \
-    pip install git+git://github.com/Theano/Theano.git && \
     pip install ipdb pytest pytest-cov python-coveralls coverage==3.7.1 pytest-xdist pep8 pytest-pep8 pydot_ng && \
```

Comment:
This can be simplified as `pip install -e .[tests]` (from the Keras source directory).